mender-convert-modify

```diff
@@ -290,6 +290,15 @@ EOF
 run_and_log_cmd "sudo cp work/mender.conf.data work/rootfs/data/mender/mender.conf"
 run_and_log_cmd "sudo chmod 600 work/rootfs/data/mender/mender.conf"
 
+log_info "Installing the local mender.conf file"
+if [ ! -e resources/mender.conf ]; then
+    log_fatal "No mender.conf file found in the resources folder.\
+ Did you remember to run the bootstrap script?"
```

Comment:
Mmm. What if a user provides a custom overlay with the config file? Is it not mandatory to run one of the bootstrap scripts?

Reply:
As it is now, it will pass if a bootstrap has been run previously.

If an overlay exists for `/etc/mender/mender.conf` then the demo settings in `mender_convert_demo_conf` will have no effect, as the overlay takes presedence.

But perhaps we should have an extra check that the bootstrap has been run? :thinking: 
I'm not sure to be honest, I mean, if the `resources/mender.conf` file exists, then bootstrap has been run at least once previously.

Difficult topic